tools/protodoc/protodoc.py

```diff
@@ -402,10 +410,10 @@ def FormatProtoAsBlockComment(proto):
 
 def GenerateRst(proto_file):
   """Generate a RST representation from a FileDescriptor proto."""
-  header = FormatHeader('=', proto_file.name)
   source_code_info = SourceCodeInfo(proto_file.source_code_info)
   # Find the earliest detached comment, attribute it to file level.
   comment = source_code_info.file_level_comment
```

Comment:
Why do you still have this line?

Reply:
? Its used in the next line. I don't know what you were using it for :D